introduction.md

```diff
@@ -0,0 +1 @@
+Hello, I am Fulla, I am 29 years old, and I am training myself in git technologies
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.